conjure-java-jackson-serialization/src/main/java/com/palantir/conjure/java/serialization/ObjectMappers.java

```diff
@@ -81,7 +77,7 @@ public static CBORMapper newClientCborMapper() {
      * </ul>
```

Comment:
Do we need to instrument CBOR mappers as well?

Reply:
It's the least commonly used of the three by a large margin, so I've opted not to include instrumentation in this change. I'll send you our encoding metrics internally

Reply:
I'm fine deferring CBOR for now, we create those outside of Conjure anyway